### DISCUSSION TITLE
Sürekli red yiyorum

### DISCUSSION THREAD

**Discussion by aysegulbildirirr** [community/community#141652] [GitHub Education]
### Hi everyone,

Github educationa başvurduğumda sürekli red yiyorum.Red yeme nedenleri ise:
- Konum erişimine izin vermek için tarayıcınızı ve işletim sisteminizi yapılandırmalısınız. VPN kullanamazsınız. Okulunuz hakkındaki bilgilerimizin eksik veya yanlış olduğuna inanıyorsanız, lütfen GitHub Education destek bileti gönderirken 'Seçtiğim okulda yanlış veya eksik bilgiler var, örneğin alan adları veya kampüs konumu' seçeneğini seçin.
-Lütfen akademik bağlılık belgenizin soyadınızı [GitHub fatura bilgilerinizde](https://github.com/settings/billing/payment_information) göründüğü gibi içerdiğinden emin olun . Yalnızca tek bir yasal adınız varsa, bunu GitHub fatura bilgilerinizin hem ad hem de soyad alanlarına girin.
Bunlardır.Nasıl çözeceğimi bilmiyorum yardimci olur musunuz??

**Comment by queenofcorgis** [comment]
Hi @aysegulbildirirr , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
